src/utils/get-txs-with-retry.ts

```diff
@@ -0,0 +1,16 @@
+import { requestWithRetry } from './request-with-retry';
+
+/* eslint-disable import/prefer-default-export */
+export const getTxsWithRetry = async (txs: string) => {
```

Comment:
not being used

Reply:
yeah, right now it's not
but it look much better alternative to this pattern
<img width="531" alt="image" src="https://github.com/user-attachments/assets/99dfc01e-26f1-4d40-a104-578f26eb41bd">